Documentation/Diagnostics/PH2117.md

```diff
@@ -0,0 +1,53 @@
+# PH2117: Avoid Unnecessary Where()
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2117 |
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [AvoidUnnecessaryWhereAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidUnnecessaryWhereAnalyzer.cs)
+| CodeFix  | No |
```

Comment:
But MS offers one: https://learn.microsoft.com/en-us/visualstudio/ide/reference/simplify-linq-expression?view=vs-2022

Reply:
Added the link and made clear this is in VS only (so not VsCode)